<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Omics Mining IDG</title>
    <link href="lib/css/bootstrap.css" rel="stylesheet">
    <link rel="stylesheet" href="css/custom_index.css">


</head>

<body ">

  <!-- Wrap all page content here -->
  <div id="wrap">

    <!-- main container - required -->
    <div id='main_container'>

      <!-- start example row -->
      <div id='homepage_examples'>

        <div class='container'>

          <div class='row'>
            <p id='example_title'>Omics Mining for the IDG: Implementation Phase</p>
          </div>

          <div class='row' >

            <div class='col-xs-6 example_col'>
              <p class='example_single_title'>Jupyter Notebooks</p>
              <a href='https://maayanlab.github.io/KMC_2017-dev/#notebooks' >
                <div class='example_image_container'>
                  <img id='notebook_image' src="img/notebook_screenshot.png">
                </div>
              </a>
              <p class='example_single_description'>Using Jupyter notebooks we explored kinase, Ion Channel, and G-protein coupled receptor clusters identified based on co-expression in the CCLE. We identified common biological themes in these clusters and inferred biolocial functions for 'dark' proteins.</p>
            </div>

            <div class='col-xs-6 example_col'>
              <p class='example_single_title'>Functional Predictions</p>
              <!-- <a href='#' target='_blank' > -->
                <div class='example_image_container'>
                    <!-- <img id='CCLE_image' src="static/img/GEN3VA_no_sidebar_small.png" > -->
                </div>
              <!-- </a> -->
              <p class='example_single_description'>We used publically available RNA-seq data to infer Gene Ontology terms for 'dark' proteins. Our predictions correctly return known ontology terms as well as predict new biological functions for 'dark' proteins. </p>
            </div>

        </div>

        </div>

        <div class='container'>

          <div class='row' >

              <div class='col-xs-6 example_col'>
                <p class='example_single_title'>Protein Similarity Matrices</p>
                <a href='https://maayanlab.github.io/KMC_2017-dev' target='_blank'>
                  <div class='example_image_container'>
                    <img id='app_screenshot' src="img/app_screenshot.png">
                  </div>
                </a>
                <p class='example_single_description'>We characterized 'dark' proteins using unsupervised clustering based on several bological datasets (e.g. GTEx), which enabled us to find associations between understudied and more well known proteins. We generated interactive similarity matrices can be used to explore protein-protein associations based on gene expression and transcription factor targets.</p>
              </div>


              <div class='col-xs-6 example_col'>
                <p class='example_single_title'>TCGA Analysis</p>

                <a href='https://drive.google.com/file/d/0B46LT_Hxo_TtdE1OeUpoTFhiTkk/view' target='_blank'>
                  <div class='example_image_container'>
                    <img class='TCGA' id='' src="img/TCGA_analysis.png" >
                  </div>
                </a>

                <p class='example_single_description'>We used gene expression and patient survival data from the TCGA to investigate several 'dark' kinases identified as part of a cluster of mitosis related kinases that were co-expressed in CCLE cancer cell lines. We demonstrate that expression of the 'dark' kinase PKMYT1 is negatively associated with patient survival.</p>
              </div>


          </div>

        </div>

    </div>

  </div>

</body>

